packages/steiger/src/features/handle-error.ts

```diff
@@ -0,0 +1,48 @@
+import { readFileSync } from 'fs'
+import { join } from 'path'
+import { TSConfckParseError } from 'tsconfck'
+import { Folder } from '@steiger/types'
+
+interface ErrorContext {
+  vfs: Folder
+}
+
+interface PackageJson {
+  dependencies?: Record<string, string>
+  devDependencies?: Record<string, string>
+}
+
+const readPackageJSON = (path: string) => JSON.parse(readFileSync(join(path, 'package.json'), 'utf-8')) as PackageJson
```

Comment:
issue: it might be the case that `package.json` is in one of the parent folders, not necessarily the one where Steiger was run

suggestion: let's use `empathic` to locate the `package.json` file:

```
import * as find from 'empathic/find'
find.up('package.json', { cwd: path })
```